iquip/apps/monitor.py

```diff
@@ -108,114 +163,148 @@ def __init__(
             row, column = idx // numColumns, idx % numColumns
             self.ttlWidgets[name] = ttlWidget
             ttlWidgetLayout.addWidget(ttlWidget, row, column)
-        self.button = QPushButton("Not Overriding?", self)
-        self.button.setCheckable(True)
+        overrideButtonBox = QGroupBox("Override", self)
+        self.overrideOnButton = QPushButton("ON", self)
+        self.overrideOffButton = QPushButton("OFF", self)
         # layout
+        overrideButtonLayout = QHBoxLayout(overrideButtonBox)
+        overrideButtonLayout.addWidget(self.overrideOnButton)
+        overrideButtonLayout.addWidget(self.overrideOffButton)
         layout = QVBoxLayout(self)
         layout.addLayout(ttlWidgetLayout)
-        layout.addWidget(self.button)
+        layout.addStretch()
+        layout.addWidget(overrideButtonBox)
         # signal connection
-        self.button.clicked.connect(self.overrideChanged)
-        self.overrideChanged.connect(self._setButtonText)
+        self.overrideOnButton.clicked.connect(lambda: self.overrideChangeRequested.emit(True))
+        self.overrideOffButton.clicked.connect(lambda: self.overrideChangeRequested.emit(False))
```

Comment:
Let's use `functools.partial()` as always! It guarantees more direct binding.

Reply:
I see. I will fix it.